File: src/main/java/entities/Question.java
package entities;

public class Question {
    private long id;
    private final String question;
    private String description;
    private final String[] answers;
    private Boolean multipleCorrectAnswers;
    private String[] correctAnswers;
    private String explanation;
    private String tip;
    private String[] tags;
    private String category;
    private Enum difficulty;



    public Question(String question, String[] answers) {
        this.question = question;
        this.answers = answers;
    }
}

File: frontend/quizmaster-frontend/src/app/question.ts
import {Difficulty} from './difficulty.enum'
import {Answer} from "./answer";
import {Tag} from "./tag";

export class Question {
  id: number | undefined;
  question = '';
  description = '';
  answers: Answer[] = [];
  multipleCorrectAnswers: boolean | undefined;
  correctAnswers: string[] = [];
  correctAnswer= '';
  explanation = '';
  tip = '';
  tags: Tag[] = [];
  category = '';
  difficulty: Difficulty | undefined;
}
